src/_guides/language/effective-dart/design.md

```diff
@@ -716,13 +735,15 @@ With types, all of this is clarified.
 The type system infers `void` for all setters automatically.
```

Comment:
Is this text aimed at Dart 2 or Dart 1? 
(Because this line was incorrect for Dart 1, but the code still uses `new` in the `Point` example above, and many other places too, which it shouldn't for Dart 2).

Reply:
Dart 1

> Because this line was incorrect for Dart 1

Which line, this one? "The type system infers `void` for all setters automatically."?

Reply:
Yes. Dart 1 had no inference. The actual return type would be `dynamic` if you didn't write anything.

Reply:
Ah, of course, when interpreting "infers" for Dart 1, I took it non-technically in the sense that Dart 1 would assume the return type to be `void`, but it being `dynamic` makes more sense.

For Dart 1, how about if we just drop the prose? @kwalrath, any suggestions?

How are the semantics different in Dart 2?

Reply:
I just checked and this guideline was added via #220 by @kwalrath and approved by @munificent. So this was meant to be for Dart 2 only?

Reply:
Hmm, #220 was done in January of this year. Dart was still at 1.21 back then.

Reply:
It fixed bug #131, which was requested by a readability reviewer. The wording might not be correct, but it reflects Dart practices within Google.

Reply:
I'm fine with dropping the explanatory prose, as long as we keep the guideline.

Reply:
Dropping the prose but keeping the guideline.